Extensions/3D/JsExtension.js

```diff
@@ -2385,9 +2385,233 @@ module.exports = {
       }
     }
 
+    class Model3DRendered3DInstance extends Rendered3DInstance {
+      constructor(
+        project,
+        layout,
+        instance,
+        associatedObjectConfiguration,
+        pixiContainer,
+        threeGroup,
+        pixiResourcesLoader
+      ) {
+        super(
+          project,
+          layout,
+          instance,
+          associatedObjectConfiguration,
+          pixiContainer,
+          threeGroup,
+          pixiResourcesLoader
+        );
+        /**
+         * Name of the resource that is rendered.
+         * If no face is visible, this will be null.
+         */
+        this._renderedResourceName = undefined;
+        const properties = associatedObjectConfiguration.getProperties();
+        this._defaultWidth = parseFloat(properties.get('width').getValue());
+        this._defaultHeight = parseFloat(properties.get('height').getValue());
+        this._defaultDepth = parseFloat(properties.get('depth').getValue());
+        const rotationX = parseFloat(properties.get('rotationX').getValue());
+        const rotationY = parseFloat(properties.get('rotationY').getValue());
+        const rotationZ = parseFloat(properties.get('rotationZ').getValue());
+        const keepAspectRatio =
+          properties.get('keepAspectRatio').getValue() === 'true';
+        const modelResourceName = properties
+          .get('modelResourceName')
+          .getValue();
+
+        this._pixiObject = new PIXI.Graphics();
+        this._pixiContainer.addChild(this._pixiObject);
+
+        this._threeObject = new THREE.Group();
+        this._threeObject.rotation.order = 'ZYX';
+        this._threeGroup.add(this._threeObject);
+
+        this._pixiResourcesLoader
+          .get3DModel(project, modelResourceName)
```

Comment:
What if `modelResourceName` is changed? Should you check this in `update` and redo a get3DModel?

Reply:
Answering to myself, this is covered by:
<img width="673" alt="image" src="https://github.com/4ian/GDevelop/assets/1280130/c293036c-1c22-4a83-a5dd-3c601bd15937">

which makes me think we should probably adapt this to also reload other kinds of resources. For now it should work.